backend/scripts/seed_data.py

```diff
@@ -1,30 +1,27 @@
+#!/usr/bin/env python
+
 # ruff: noqa: E501 Line too long
 
 from textwrap import dedent
 
 import requests
 from fastapi.testclient import TestClient
+from sqlalchemy.orm import Session
 
 from test_observer.controllers.test_executions.models import (
     C3TestResult,
     C3TestResultStatus,
     EndTestExecutionRequest,
     StartTestExecutionRequest,
 )
-from test_observer.controllers.users.models import CreateUserRequest
 from test_observer.data_access.models_enums import FamilyName
+from test_observer.users.add_user import add_user
 
 BASE_URL = "http://localhost:30000/v1"
 CREATE_USER_URL = f"{BASE_URL}/users"
```

Comment:
Just a small nitpick, maybe we can remove this variable as it is no longer used.

Reply:
I'm surprised that ruff didn't complain about it